pyproject.toml

```
@@ -48,6 +48,7 @@ beautifulsoup4 = "^4.12.3"
 pandas = "^2.1.4"
 marko = "^2.1.2"
 openpyxl = "^3.1.5"
+ocrmac = { version = "^1.0.0", optional = true }
```

I think it is still better to have the marker for macos here

```suggestion
ocrmac = { version = "^1.0.0", markers = "sys_platform == 'darwin'", optional = true }
```

---

@nuridol I think this is really the last small requirements 👍

---

Thank you for your suggestion. I have updated the files accordingly.